src/OEUnit/Automation/Pct/RunTests.p

```diff
@@ -73,42 +76,35 @@ END PROCEDURE.
 
 
   /*----------------------------------------------------------------------------
-    Searches recursively for class files in a given path. Full filenames are 
+    Searches recursively for class files in a given path. Full filenames are
     returned seperated by a star(*)
-  ----------------------------------------------------------------------------*/  
+  ----------------------------------------------------------------------------*/
 PROCEDURE FindClassFiles PRIVATE:
   DEFINE INPUT PARAMETER path AS CHARACTER NO-UNDO.
-  DEFINE OUTPUT PARAMETER classFiles AS CHARACTER NO-UNDO INIT "".
 
-  DEFINE VARIABLE childClassFiles AS LONGCHAR NO-UNDO.
   DEFINE VARIABLE directoryEntry AS CHARACTER NO-UNDO.
-    
+  
   INPUT FROM OS-DIR (path).
   REPEAT:
-        
+
     IMPORT directoryEntry.
     FILE-INFO:FILE-NAME = path
       + (IF OPSYS = "WIN32" THEN "\" ELSE "/")
       + directoryEntry.
-    
+
     CASE SUBSTRING(FILE-INFO:FILE-TYPE, 1, 1):
       WHEN "F" THEN DO:
-        IF directoryEntry MATCHES ("*.cls") THEN
-          classFiles = classFiles
-            + (IF classFiles <> "" THEN "*" ELSE "")
-            + FILE-INFO:FULL-PATHNAME.
+        IF directoryEntry MATCHES ("Test*.cls") THEN DO:            
```

Comment:
There is a change here in the functionality - previously was identifying "*.cls", and is now identifying only files that match "Test*.cls".